pytest_fixtures/core/broker.py

```diff
@@ -29,7 +29,7 @@ def _target_sat_imp(request, _default_sat, satellite_factory):
     if request.node.get_closest_marker(name='destructive'):
         new_sat = satellite_factory()
         yield new_sat
-        VMBroker(hosts=[new_sat]).checkin()
+        Broker(hosts=[new_sat]).checkin()
```

Comment:
Could you please include a step to unregister sat_host before checkin?
```suggestion
        new_sat.unregister()
        Broker(hosts=[new_sat]).checkin()
```